src/Application/Channels/Queries/ChannelExtensions.cs

```diff
@@ -0,0 +1,22 @@
+﻿
+using Hippo.Core.Entities;
+
+namespace Hippo.Application.Channels.Queries;
+
+public static class ChannelExtensions
+{
+    public static ChannelSummaryDto ToChannelSummaryDto(this Channel channel)
```

Comment:
Currently, when the App list page is loaded, each App's direct mapping is brought, meaning we also load each app's revisions, channels and each channel's active revision. We thought this would not be the most efficient way of doing it since on the app list page we will only show the App names and channel statuses.

This is also the reason I did not use the AutoMapper for the AppDto (since Channel and ChannelSummaryDto have different structures). There might be a way to configure it to map from nested objects (eg. from Channel.ActiveRevision.RevisionNumber to ChannelSummaryDto.ActiveRevisionNumber) as well but we did not dive into that yet